---
layout: page_en
title: Pairwise Alignment and Remove Gaps - Unipro UGENE
---

<h1>
    UGENE Tips: Pairwise Alignment, Remove Gaps and more<br/>
</h1>
<h3>
    Pairwise Alignment<br/>
</h3>
<p>
    <a href="/assets/images/learn/14.jpg"><img loading="lazy" alt="Pairwise Alignment"
                                               class="aligncenter size-full wp-image-3056"
                                               height="751"
                                               src="/assets/images/learn/14.jpg"
                                               width="1357"/></a>
</p>
<p>
    Did you know how to build quickly a pairwise alignment for sequences included to a multiple alignment?
</p>
<p>
    To do this you need to open the &quot;Pairwise Alignment&quot; tab in the Alignment Editor Options Panel and specify two sequences there. The
    following algorithms are available for producing a pairwise alignment &#8211; Smith-Waterman algorithm and KAlign algorithm. Pairwise alignment
    results are saved to a separate file by default, but they also can be saved into the original file.
</p>
<p>
    On attached screenshots you can see an example of how to build a pairwise alignment displayed in a separate window.
</p>
<p>
    See more about the work with pairwise alignments in <a href="https://doc.ugene.net/wiki/display/UM{{site.ugene.doc_version}}/Pairwise+Aligning">our
    documentation</a>.
</p>
<h3>
    Distance<br/>
</h3>
<p>
    <a href="/assets/images/learn/15.jpg"><img loading="lazy" alt="Distance"
                                               class="aligncenter size-full wp-image-3057"
                                               height="909"
                                               src="/assets/images/learn/15.jpg"
                                               width="1428"/></a>
</p>
<p>
    Did you know how to get quickly distance statistics about sequences included to a multiple alignment?
</p>
<p>
    To do this you need to start with setting a reference sequence which others are going to be compared with. Then you should open the &quot;Statistics&quot;
    tab on the Options panel and set the &quot;Show distances column&quot; checkbox. After that your aligned sequences list is supplemented with a
    distance values column.
</p>
<p>
    On attached screenshots you can see an example of how to show distances for a chosen reference sequence.
</p>
<p>
    See more about displaying sequence statistics in the Multiple alignment editor in <a
                                                                                         href="https://doc.ugene.net/wiki/display/UM{{site.ugene.doc_version}}/Statistics">our
    documentation</a>.
</p>
<h3>
    Remove Gaps<br/>
</h3>
<p>
    <a href="/assets/images/learn/16.jpg"><img loading="lazy" alt="Remove Gaps"
                                               class="aligncenter size-full wp-image-3058"
                                               height="909"
                                               src="/assets/images/learn/16.jpg"
                                               width="1428"/></a>
</p>
<p>
    Did you know how to delete swiftly columns with gaps from a multiple alignment?
</p>
<p>
    To do this open a needed file with a multiple alignment by UGENE. Then select &quot;Edit -&gt; Remove columns of gaps&#8230;&quot; in the
    right-click menu. After that a dialog appears allowing you to specify a number of gaps in columns to be deleted. This number can be either an
    absolute one or a percentage of a bases count in a column.
</p>
<p>
    Also there is a possibility of removing all the gaps within an alignment. In order to do this select &quot;Edit -&gt; Remove all gaps&quot; in
    the multiple alignment editor right-click menu.
</p>
<p>
    On attached screenshots you can see an example of how to delete columns more than a half of which consist of gaps from an alignment.
</p>
<p>
    See more about deleting gaps from an alignment in <a
                                                         href="https://doc.ugene.net/wiki/display/UM{{site.ugene.doc_version}}/Removing+Columns+of+Gaps">our
    documentation</a>.
</p>
<h3>
    Make a Phylogenetic Tree<br/>
</h3>
<p>
    <a href="/assets/images/learn/17.jpg"><img loading="lazy" alt="Make a Phylogenetic Tree"
                                               class="aligncenter size-full wp-image-3059"
                                               height="752"
                                               src="/assets/images/learn/17.jpg"
                                               width="1310"/></a>
</p>
<p>
    Did you know how to make <a href="http://ugene.unipro.ru/%E2%80%8Bugene-phylogenetic-tree-algorithms/">a phylogenetic tree</a>
    by a given multiple alignment?
</p>
<p>
    To do this open a needed file with a multiple alignment by UGENE. Then expand the &quot;Tree Settings&quot; tab on the Options panel and click
    &quot;Build tree&quot;. Then a dialog should appear in which you can specify a tree calculation algorithm and a path to a file which the tree is
    saved in. By default the tree is displayed in the same window as the multiple alignment. Alternatively, it can be shown in a separate window.
</p>
<p>
    On attached screenshots you can see an example of how to make a phylogenetic tree for aligned 16S rRNA sequences of several strains of
    cyanobacteria.
</p>
<p>
    See more about available tree calculation algorithms in <a
                                                               href="https://doc.ugene.net/wiki/display/UM{{site.ugene.doc_version}}/Phylogenetic+Tree+Viewer">our
    documentation</a>.</p>
